crates/sui/tests/move_test_code/sources/quasi_shared_objects.move

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
We already have Move integration tests in `sui-core` that look similar to these (https://github.com/MystenLabs/sui/blob/main/crates/sui-core/src/unit_tests/move_integration_tests.rs). Is there a reason why these are separate or should we keep them in the same place?

Reply:
I could put them there, sure